patches/bpf.h.patch

```diff
@@ -69,5 +69,5 @@ index 71efd9a..59fdf0e 100644
 -u_int	bpf_filter_with_aux_data(const struct bpf_insn *, const u_char *, u_int, u_int, const struct bpf_aux_data *);
 +/* u_int	bpf_filter_with_aux_data(const struct bpf_insn *, const u_char *, u_int, u_int, const struct bpf_aux_data *); */
  
- 
- __END_DECLS
+ /*
+  * events to be tracked by bpf_register_track_event callbacks
```

Comment:
This file ends with an unterminated comment. Is this intentional?

Reply:
Ah, yes, its the context part that changed.
This change is needed to avoid confusion in `git apply` next time we patch.